package.json

```diff
@@ -17,10 +17,14 @@
     "argon2": "^0.28.2",
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
+    "google-maps-react": "^2.0.6",
     "jsonwebtoken": "^8.5.1",
+    "node-fetch": "^2.6.6",
     "pg": "^8.6.0",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "react-google-maps": "^9.4.5",
```

Comment:
is one of these google maps things not needed anymore? you can uninstall packages with `npm uninstall <package-name>`. e.g. `npm uninstall react-google-maps`